### PR TITLE
Tweak error messaging for graphql errors

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
**Issue:** https://github.com/department-of-veterans-affairs/va.gov-team/issues/20101

This PR attempts to improve the GraphQL error logging.

## Testing done
N/A

## Screenshots
image
image

## Acceptance criteria
- [x] Improve graphql error logging

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/stages/build/drupal/api.js [hunk 1]
```diff
@@ -220,8 +220,12 @@ function getDrupalClient(buildOptions, clientOptionsArg) {
...
 
         if (json.errors) {
-          console.log(json.errors);
-          throw new Error(`${queryName} resulted in errors`);
+          const formattedErrors = JSON.stringify(json.errors, null, 2);
+          const pluralizedErrors =
+            json.errors.length > 1 ? 'errors' : 'an error';
+          throw new Error(
+            `Error with ${queryName}:\n\n${formattedErrors}\n\nGraphQL query that has ${pluralizedErrors}:\n${query}`,
+          );
```
Comment (ncksllvn) [inline src/site/stages/build/drupal/api.js:228]: I actually wonder if we could make use of `chalk` here at all. Like maybe the error could be printed in red; the GraphQL query in blue; and then the actual error we throw is something basic like "GraphQL error encountered" or something like that

Comment (kelsonic) [reply src/site/stages/build/drupal/api.js:228]: image
image

Great suggestion, updated! 🙂

Comment (ncksllvn) [reply src/site/stages/build/drupal/api.js:230]: Whoaaaa 👏

**Review by ncksllvn — APPROVED** [review]
LGTM!

**Review by U-DON — APPROVED** [review]

**Pull request merged**